src/index.ts

```diff
@@ -53,11 +125,27 @@ export const getRequestBody = (
 
   requestBody['pd[tm]'] = payload.tm || 'pinterest-mc'
 
+  // match  event types to Pinterest's default
+  const eventTypes: Record<string, string> = {
+    'Product Added': 'addtocart',
+    'Order Completed': 'checkout',
+    'Products Searched': 'search',
+  }
+
+  if (eventTypes[eventType]) {
+    requestBody.event = eventTypes[eventType]
+  }
+
+  const ecommerceData: Record<string, string | number> | null =
```

Comment:
I believe this type annotation is redundant, as TS can easily deduce the type from the function being called
```suggestion
  const ecommerceData =
```

Reply:
It actually throws an error if I make this change, so I kept it this way.